File: Responder/Responder/Commands/LaunchAppCommand.cs
using System;
using System.Collections.Generic;

namespace Responder.AppConfig
{
    public class LaunchAppCommand : ICommand
    {
        public CommandType Type { get; private set; }
        public int Length { get; set; }
        public int AppId { get; set; }
        public String Argument { get; set; }

        public LaunchAppCommand()
        {
            Type = CommandType.LaunchApplication;
        }
    }
}

File: Imperium/src/com/imperium/Commands/LaunchAppCommand.java
package com.imperium.Commands;

public class LaunchAppCommand implements ICommand {
    private CommandType type;
    private int length;
    private int appId;
    private String argument;

    public CommandType getType() {
        return type;
    }

    public int getLength() {
        return length;
    }

    public void setLength(int length) {
        this.length = length;
    }

    public int getAppId() {
        return appId;
    }

    public void setAppId(int appId) {
        this.appId = appId;
    }

    public String getArgument() {
        return argument;
    }

    public void setArgument(String argument) {
        this.argument = argument;
    }

    public LaunchAppCommand(){
        type = CommandType.LaunchApplication;
    }
}
